site/public/js/forum.js

```diff
@@ -946,6 +954,17 @@ function showEditPostForm(post_id, thread_id, shouldEditThread, render_markdown,
         },
     });
 }
+// eslint-disable-next-line no-unused-vars
+function markForDeletion(ele) {
```

Comment:
It would be better to add `/* exported markForDeletion */` to the top of the file rather than adding inline ignores for everything, as per our [documentation](https://submitty.org/developer/software_and_system_design/coding_style_guide/javascript#style-rules).